prowler/providers/azure/services/vm/vm_ensure_disks_encrypted_with_cmk/vm_ensure_disks_encrypted_with_cmk.py

```diff
@@ -0,0 +1,25 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.vm.vm_client import vm_client
+
+
+class vm_ensure_disks_encrypted_with_cmk(Check):
```

Comment:
how about change the check name to `vm_ensure_attached_disks_encrypted_with_cmk`?